google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateClusterRequest.java

```diff
@@ -84,13 +84,60 @@ public CreateClusterRequest setZone(String zone) {
   /**
    * Sets the number of nodes allocated to this cluster. More nodes enable higher throughput and
    * more consistent performance.
+   *
+   * @deprecated Please use CreateClusterRequest#setScalingMode instead
    */
   @SuppressWarnings("WeakerAccess")
+  @Deprecated
   public CreateClusterRequest setServeNodes(int numNodes) {
     proto.getClusterBuilder().setServeNodes(numNodes);
     return this;
   }
 
+  /**
+   * Sets the scaling node to manual and sets the number of nodes allocated to this cluster. More
+   * nodes enable higher throughput and more consistent performance.
+   */
+  public CreateClusterRequest setScalingMode(StaticClusterSize staticClusterSize) {
```

Comment:
Do we need `@Nonnull StaticClusterSize staticClusterSize` and `@Nonnull ClusterAutoscalingConfig autoscalingConfig` or check if `staticClusterSize` and `autoscalingConfig` is null first?

Reply:
thanks!